static-web/css/quiz.css

```diff
@@ -0,0 +1,138 @@
+/*
```

Comment:
Never put a comment before the doctype. The browser needs to know it's dealing with HTML before anything else.